plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -292,11 +293,11 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
         val linuxExec = target.createTask<LinuxExec>("setupLinuxExecEnvironment") { task ->
             task.dependsOn(copyGraalVM)
             task.baseDirectory.set(target.projectDir.toPath().toString())
-            task.tag.set("conclave-build:$CONCLAVE_SDK_VERSION")
+            task.tag.set("conclave-docker-dev.software.r3.com/com.r3.conclave/conclave-build:$DOCKER_CONCLAVE_BUILD_TAG")
             // Create a 'latest' tag too so users can follow our tutorial documentation using the
             // tag 'conclave-build:latest' rather than looking up the conclave version.
-            task.tagLatest.set("conclave-build:latest")
-            task.buildInDocker.set(conclaveExtension.buildInDocker)
+            task.tagLatest.set("conclave-docker-dev.software.r3.com/com.r3.conclave/conclave-build:latest")
```

Comment:
Can we get rid of this. It's not correct as we are passing in the correct tag for the user automatically.